README.md

```diff
@@ -0,0 +1,17 @@
+# CI Workflows
+
+This repository contains GitHub Actions workflows used by various IPLD repositories.
+By storing them in a central place (here), and distributing them in an automated way, we solve multiple problems:
+1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
+2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
+
+## Technical Details
+
+This repository currently defines two workflows for Go repositories:
+* [go-check](https://github.com/ipld/.github/blob/master/workflow-templates/go-check.yml): Performs checks that ensure that the code adheres to common coding styles.
+* [go-test](https://github.com/ipld/.github/blob/master/workflow-templates/go-test.yml): Runs unit tests, using different compiler versions and operating systems.
```

Comment:
nit: `go test` may be about more than unit tests :) I think you should just say "runs all tests".